src/common/event-utils/index.js

```diff
@@ -24,4 +24,35 @@ function handleUpDownArrowsKeydown(e, callback) {
     handleKeydown([38, 40], e, callback);
 }
 
-module.exports = { handleActionKeydown, handleEscapeKeydown, handleUpDownArrowsKeydown };
+const handlers = [];
+function addEventListener(_, handler) {
```

Comment:
What is the underscore for?

Reply:
We don't need it but the `subscribeTo` takes in 2 arguments hence the ` _`